.github/workflows/prod-build-deploy-cloudflare.yml

```diff
@@ -0,0 +1,70 @@
+name: Prod Build and Deploy (CloudFlare)
+
+on:
+  push:
+    branches: ['main']
+  workflow_dispatch:
+  pull_request:
+
+env:
+  GOOGLE_ANALYTICS_TRACKING_ID: ${{ secrets.GOOGLE_TAGMANAGER_ID_SCDF }}
+
+permissions:
+  contents: read
+  pull-requests: write
+
+jobs:
+  # build and deploy staging
+  build:
+    name: Build site
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      # agent may give node which doesn't work for us
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18.2.0
+
+      # setup cf cli with bg plugin and login
+      # - uses: jvalkeal/setup-cf-cli@v0
+      #   with:
+      #     version: 6.51.0
+      #     plugin-repo: https://plugins.cloudfoundry.org
+      #     plugin-id: blue-green-deploy
+      #     login: true
+
+      # do site build and prepare for cf push
```

Comment:
Ha! I am guessing "cf push" here means "CloudFlare push" and not the predecessor well-know "cf push" for CloudFoundry.